app/views/achievements/_form.html.haml

```diff
@@ -10,17 +10,17 @@
         = f.text_area :description, class: "form-control"
     .form-group
       = f.label :points, t(:achievement_points), class: ["col-sm-2", "control-label"]
-      .col-sm-10
-        = f.text_field :points, class: "form-control"
+      .col-sm-2
+        = f.text_field :points, class: "form-control", type: "number", min: 1, max: 3, size: 5
     .form-group
       = f.label :side, t(:achievement_side), class: ["col-sm-2", "control-label"]
-      .col-sm-10
+      .col-sm-2
         = f.select :side, side_options, {}, class: "form-control"
     .form-group
       .col-sm-offset-2.col-sm-10
-        = button_tag class: ["btn", "btn-success"] do
-          %span.glyphicon.glyphicon-pencil
-          = t(:save_achievement)
-        %a.btn.btn-danger{ href: achievements_path }
+        %a.btn{ href: manage_achievements_path }
           %span.glyphicon.glyphicon-remove
           = t(:cancel)
+        = button_tag class: ["btn", "btn-primary"] do
+          %span.glyphicon.glyphicon-pencil
+          = t(:save_achievement)
```

Comment:
Swapped the order of these buttons only because I generally expect the `cancel` action to be to the left of the `primary` action.

I'm sure I've read a UX style guide about this convention: [here's one.](http://uxmovement.com/buttons/why-ok-buttons-in-dialog-boxes-work-best-on-the-right/) It is just a convention rather than a rule so if you don't dig that I'll amend my commit. (If you do okay it then I'll update the rest of the site to follow suit for consistency.)